src/OpenTelemetry/Sdk.cs

```diff
@@ -32,7 +34,24 @@ namespace OpenTelemetry
     /// </summary>
     public static class Sdk
     {
-        private static TimeSpan defaultPushInterval = TimeSpan.FromSeconds(60);
+        private static readonly TimeSpan DefaultPushInterval = TimeSpan.FromSeconds(60);
+
+        private static readonly RuntimeContextSlot<bool> SuppressInstrumentationRuntimeContextSlot = RuntimeContext.RegisterSlot<bool>("otel.suppress_instrumentation");
```

Comment:
This would help the exporters to quickly look up the suppress_instrumentation flag without a hash lookup.